io/io/src/TBufferMerger.cxx

```diff
@@ -280,6 +280,7 @@ TBufferMerger::~TBufferMerger()
    fCV.notify_one();
 
    fMergingThread->join();
+   fFile->Close();
```

Comment:
No, fFile is owned by the TBufferMerger class, it's a unique_ptr<TFile>. It gets destroyed with the server. I just noticed, however, that the documentation on the header is wrong, I will update it.